price/priceService.ts

```diff
@@ -0,0 +1,34 @@
+import { baseUrl } from './url.js';
+import {Pair} from '../types/common.js'
+
+type TimeFrame = string;
+
+interface Candle {
+  startedAt: string,
+  ticker: string,
+  resolution: "1MIN"|"5MINS"|"15MINS"|"30MINS"|"1HOUR"|"4HOURS"|"1DAY",
```

Comment:
Prettier should be automatically applied on this file, and put whitespaces between these union items. I wonder why it didn't.  
Please check the issue with @Mr-MRF-Dev, and make sure git hooks are run when you try to commit.

Reply:
okay I asked Mohammad and check this